tests/fuzz/runFuzzTest.sh

```diff
@@ -22,14 +22,15 @@ usage() {
     exit 1
 }
 
-if [ "$#" -ne "4" ]; then
+if [ "$#" -ne "5" ]; then
     usage
 fi
 
 TEST_NAME=$1
 FUZZ_TIMEOUT_SEC=$2
-CORPUS_UPLOAD_LOC=$3
-ARTIFACT_UPLOAD_LOC=$4
+BUILD_DIR_PATH=$3
```

Comment:
Are these arguments going to be updated in a future PR? or rather, can I review how this script is being called somewhere ?

Reply:
Only in CMakeList.txt? No existing codebuild job is calling it?

Reply:
This is only ran in CMake after we migrated away from make. codebuild jobs will indirectly call it through cmake.